### PR TITLE
HW2 (ea2905) Added custoplayer and jotai project proposals

### PR DESCRIPTION
**Pull request by Etesam913**
✨ Added jotai.md proposal

🔗 Added links to proposals

📝 Added extra specificity on jotai proposal

🚒 Moved project into projects/javascript 📁

🔗 Changed proposal links from absolute to relative

🔗 Added repo link to custoplayer proposal

### CODE REVIEW

File: students/ea2905.md [hunk 1]
```diff
@@ -4,17 +4,13 @@ Name: Etesam Ansari
...
+![GitHub](https://github.com/etesam913)
 
-## Project Proposal
-
-During much of 2022 I was working on a personal project to help people create customizable video players. The goal was to create a npm React.js library that exposes a React.js component. This component allows users to make changes (ex: change colors, change icons, change layouts) to the video player to make it customizable.
+![GitHub README](https://github.com/Etesam913/etesam913/blob/master/README.md)
 
-I want to kind of do a rewrite of this project as there were many bad design decisions that I made when first coding it up. For example, I want to use Typescript (for better intellisense and devX in general) and use a third party state management library instead of React Context.
+## Project Proposal
+![Proposal](projects/javascript/custoplayer.md)
```
Comment (timkpaine) [inline students/ea2905.md:12]: update link path and re-squash 😈

File: students/ea2905.md [hunk 2]
```diff
@@ -4,17 +4,13 @@ Name: Etesam Ansari
...
 
 ## Project Contributions
+![Contributions](projects/javascript/jotai.md)
```
Comment (timkpaine) [inline students/ea2905.md:15]: update link path and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: students/ea2905.md (hunk 1 above)
Comment (Etesam913) [reply students/ea2905.md]: I fixed the link path

File: students/ea2905.md (hunk 2 above)
Comment (Etesam913) [reply students/ea2905.md]: I fixed the link path

**Review by timkpaine — APPROVED** [review]
lgtm

**Pull request merged**